### DISCUSSION TITLE
怎么把代码块单独设置成黑色主题呢？

### DISCUSSION THREAD

**Discussion by zeng-yq** [ppoffice/hexo-theme-icarus#977] [Q&A] [answered]
代码块默认是白色的，想要黑色的

**Comment by eldicn** [comment, marked as answer]
https://github.com/ppoffice/hexo-theme-icarus/blob/3264e1859e724804b46e37e62c82e7862d89c9d0/_config.icarus.yml#L121-L130

修改配置文件中的 `theme` 字段为喜欢的主题

[▶ 点击预览所有主题](https://highlightjs.org/static/demo/)

比如 `Atom One Dark`，打开[此链接](https://github.com/highlightjs/highlight.js/tree/main/src/styles)找到对应的 CSS 名称 `atom-one-dark`

**Reply by zeng-yq** [reply]
感谢，已解决
